source/command-monitoring/tests/README.rst

```diff
@@ -42,5 +42,5 @@ Additional Values
 
 The expected events provide the minimum data that is required and can be tested. It is
 possible for more values to be present in the events, such as extra data provided when
-using sharded clusters. The driver MUST assert the expected data is present and also
-MUST allow for additional data to be present as well.
+using sharded clusters or ``nModified`` fields in updates. The driver MUST assert the
```

Comment:
Oy.  I see now.  Sorry.  So the only small issue is that "fields" is plural, but there is only one field (nModified) referenced.  So instead how about

     or the ``nModified`` field in the update command response.

?